.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Code Quality
```

Comment:
Is this workflow just for linting? I think we can rename the file also. and remove the last build step :thinking:

Reply:
nvm I guess is just a check to see if it builds xd, its okay then

Reply:
por qué hablas en inglés? jaja si mi descripción ta en español mal trip but ok

Reply:
lol

Reply:
Fue automatico ajsdfasdfasdfjasdf, la costumbre